src/components/cms/Dashboard/TableLoader.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Put this TableLoader in the shared folder and try to use it at all places where table is used.

Reply:
Yes Sir , I have already  used it in all places possible